app/signer/bls.go

```diff
@@ -507,3 +512,49 @@ func CreateBlsKey(homeDir string, password string, passwordFilePath string, cmd
 
 	return nil
 }
+
+// UpdateBlsPassword updates the password for a BLS key
+// Takes a password that was determined by the password determination logic.
+func UpdateBlsPassword(homeDir string, blsPrivKey bls12381.PrivateKey, password string, passwordFilePath string, cmd *cobra.Command) error {
```

Comment:
I thought this would remove the `passwordFilePath` from the Bls key structure
@gitferry

Reply:
I don't understand. The `passwordFilePath` is filled in `NewBls`, no?